lib/internal/util/inspect.js

```diff
@@ -91,10 +93,19 @@ const assert = require('internal/assert');
 
 const { NativeModule } = require('internal/bootstrap/loaders');
 
+const setSizeGetter = uncurryThis(
+  Object.getOwnPropertyDescriptor(SetPrototype, 'size').get);
+const mapSizeGetter = uncurryThis(
+  Object.getOwnPropertyDescriptor(MapPrototype, 'size').get);
+const setValues = uncurryThis(
+  Object.getOwnPropertyDescriptor(SafeSet.prototype, 'values').value);
+const mapEntries = uncurryThis(
+  Object.getOwnPropertyDescriptor(SafeMap.prototype, 'entries').value);
```

Comment:
I first wanted to use `MapPrototype` but that did not work. I did not look into why it did not work. Does someone know why we handle some properties special? @devsnek @targos @joyeecheung maybe?


```
internal/per_context/primordials.js:24
  return (thisArg, ...args) => ReflectApply(func, thisArg, args);
                               ^

TypeError: Method Map.prototype.entries called on incompatible receiver undefined
    at entries (<anonymous>)
    at Map.<anonymous> (internal/per_context/primordials.js:24:32)
```

Reply:
I will look into it when I'm not mobile. Unrelatedly, I think you could remove the property descriptor lookup for entries.